docs/doc/30-reference/30-sql/00-ddl/70-clusterkey/dml-set-cluster-key.md

```diff
@@ -0,0 +1,26 @@
+---
+title: SET CLUSTER KEY
+description:
+  Sets a cluster key when creating a table.
+---
+
+Sets a cluster key when creating a table.
```

Comment:
We should introduce what the CLUSTER KEY will do if you add it.

Reply:
updated.